recorder/recorder.go

```diff
@@ -448,17 +487,27 @@ func (rec *Recorder) SetRealTransport(t http.RoundTripper) {
 	rec.options.RealTransport = t
 }
 
+// Block unsafe methods from ever being called with RealTransport.
+// The definition of "Safe Methods" comes from
+// https://datatracker.ietf.org/doc/html/rfc2616#section-9.1.1
```

Comment:
Please change the reference to RFC 9110 here instead (details in the previous comment)